Automattic/wp-calypso
# Replace `npm` commands with `yarn` equivalents

#### Changes proposed in this Pull Request

* Updates all invocations to `npm` across the repo with their `yarn` equivalent.

#### Testing instructions

* Clone the branch and run a few commands (eg: `yarn start`) and make sure they still work.
